src/confd/share/test.d/10-system.json

```diff
@@ -0,0 +1,14 @@
+{
```

Comment:
Symlink this file into the file in factory config instead of copy. But it can most likely go into the gen-hostname script, since the password is now static ($factory$).